docs/pages/access-controls/idps/saml-guide.mdx

```diff
@@ -18,56 +18,105 @@ authenticate to external services.
   Reference](./saml-reference.mdx) before proceeding.
 - User with permission to create service provider resource. The preset `editor` role has this permission.
 - SAML application (also known as a SAML service provider or SP) for testing. For this guide, we'll be using
-[samltest.id](https://samltest.id/) as a service provider for Teleport's SAML IdP.
+[RSA Simple Test Service Provider](https://sptest.iamshowcase.com/) (iamshowcase app) - a free test service that
+let's us test Teleport SAML IdP. The test service has a protected page, which can be accessed only after a
```

Comment:
I don't see the term "iamshowcase app" on the site itself. Can we call it "Simple Test Service Provider"?

Reply:
Updated to refer full name "Simple Test Service Provider" in 
https://github.com/gravitational/teleport/pull/42337/commits/597b995df63cabf2329c0023bd36a68fd90c3044.

The SAML metadata of the site refers "iamshowcase" as entity ID, which is the canonical name of the service provider.